### ISSUE TITLE
Folder selection overlays the keyboard

### ISSUE THREAD

**Issue by shanks1337** [syncthing/syncthing-android#1877]
Folder selection overlays the keyboard

![Screenshot_20221216-025756](https://user-images.githubusercontent.com/8984284/210120742-33cf928b-75b2-4475-98de-149c76efda0d.png)
![Screenshot_20221216-025912](https://user-images.githubusercontent.com/8984284/210120743-f8586f47-3c68-4119-b2f1-470c81c007ab.png)

Using the back gesture to close the keyboard won't close the overlay

### Version Information

    App Version: 1.22.2
    Syncthing Version: v1.22.2
    Android Version: Android 13

**Comment by tomasz1986** [comment]
Is this stock(ish) Android? I can actually reproduce the problem on a Samsung device running Android 12 but it would be good to know whether this affects Android in general or not. Because this is about the Web GUI itself and not the Android app, the issue belongs to https://github.com/syncthing/syncthing/issues really, not to this repository.

However, I'm not sure whether this alone is worth having a separate issue though (as these kind of small glitches are nothing new when it comes to the Web GUI 😛). Honestly, as long as things are still usable, I wouldn't bother too much, unless you're going to make a pull request and fix the problem yourself.

**Comment by shanks1337** [comment]
This was tested on a Google Pixel 7, so yeah stock(ish) / vanilla Android + the Pixel specific features.

It's useable but it makes it more difficult. Sometimes the back button can barly be reached or the first row of keyboard button is not available due the cover of the overlay. As I mentioned, the back gesture which closes the keyboard won't close the path suggestions as well. Using the back gesture again brings you back to previous screen. I include two more screenshots to maybe illustrate the problem a little bit better when more path options are available.

![Screenshot_20221231-024735](https://user-images.githubusercontent.com/8984284/210172812-1e63c185-8707-4e5b-9042-9115bf7c52ff.png)
![Screenshot_20221231-024659](https://user-images.githubusercontent.com/8984284/210172815-c91cf6a7-daeb-493c-a67f-1eb22eb42f10.png)

**Comment by tomasz1986** [comment]
Yeah, the overlay positioning seems somewhat broken, and also it would probably be good to limit the number of entries displayed on the screen (with a possibility to scroll the list when needed). I may have a look at this later on.

**Comment by wweich** [comment]
I'm pretty sure, this isn't actually our overlay, but a "normal" drop down list, rendered by Android. The position in the first screenshots however suggest, that we (or the old Bootstrap version we use) don't position the invisible drop down field correctly.
